hms-app/src/components/RelevantIdeasLoader.tsx

```diff
@@ -101,9 +101,11 @@ export default function RelevantIdeasLoader({
     }
   }, [ideaData])
 
-  setRelevantIdea(relevantIdeaList)
-  setHackathon(hackathonData)
-  setLoading(isLoading)
+  useEffect(() => {
```

Comment:
it might be that they should all be in different `useEffect` calls :thinking: but I'm not sure honestly.  Like this is `isLoading` changes it will cause all of the `setXxx()` to be called, but I really don't know which is better

Reply:
This should be fine as:
`useEffect(() => {
  //Runs on initial render and any time any dependency value change
}, [dep1, dep2]);`